docs/modules/ROOT/pages/how-to-guides/Secure-your-supply-chain/proc_java_dependencies.adoc

```diff
@@ -290,7 +370,7 @@ legacyRepos: <6>
 After adding this information, re-running the build should resolve this information, moving it to the  state `ArtifactBuildBuilding`, and eventually to `ArtifactBuildComplete`.
 
 
-==== Identifying why a build failed [[failed_builds]]
+=== Identifying why a build failed [[failed_builds]]
```

Comment:
we already have a section up there that we're gonna call "Troubleshooting failed builds". move this content there?